cookbooks/inner_core_convection/inner_core_convection.cc

```diff
@@ -111,6 +157,39 @@ namespace aspect
             Functions::ParsedFunction<dim>::declare_parameters (prm, 1);
           }
           prm.leave_subsection();
+
+          prm.declare_entry ("Phase transition depth", "0.0",
```

Comment:
do the inner core people state locations in terms of depth from the surface, or in terms of distance from the center?

Reply:
I asked, and it looks like the modellers state locations in terms of distance from the centre, and the seismologists in terms of distance from the inner core boundary. I changed everything to use the radius instead of depth, which is useful anyway, as eventually we want to let the core grow over time...